l10n_it_account_balance_report/models/account_type.py

```diff
@@ -0,0 +1,15 @@
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import fields, models
+
+
+class AccountType(models.Model):
+    _inherit = 'account.account.type'
+
+    account_balance_report_section = fields.Selection(
```

Comment:
Come mai non usare il campo esistente `internal_group`?

Reply:
@eLBati mi era stato detto così allo sprint napoletano di giugno, non essendo un esperto di contabilità mi son fidato 😅

Reply:
Ok però bisogna trovare la risposta

Reply:
@alessandrocamilli tu sai dirci di più?

Reply:
Il campo serve per definire se il conto va stampato tra i costi o ricavi del conto economico, oppure se tra le attività o passività dello stato patrimoniale. Il campo internal_group definisce questo? Altrimenti lasciamo così com'è

Reply:
Il campo `internal_group` può assumere i seguenti valori:
```
        ('equity', 'Equity'),
        ('asset', 'Asset'),
        ('liability', 'Liability'),
        ('income', 'Income'),
        ('expense', 'Expense'),
```
quindi molto simili a `account_balance_report_section` (a parte `equity` e `asset` che per `account_balance_report_section` si tratta sempre di attività).

Comunque se vedete problemi nell'utilizzare `internal_group`, vi chiedo solo di scriverlo in un commento sopra la definizione di `account_balance_report_section`

Reply:
io non userei internal_group per lasciare la stampa indipendente da quelle che possono essere le definizioni contabili di Odoo. @SilvioGregorini puoi scrivere questo?

Reply:
@eLBati @alessandrocamilli ho aggiornato la PR e, contestualmente, anche la PR su 11.0: https://github.com/OCA/l10n-italy/pull/1282

Reply:
ok, grazie